assets/js/services/auth_service.js

```diff
@@ -1,26 +1,42 @@
-app.factory("AuthService", ["$resource", "$http", function($resource, $http) {
+app.factory("AuthService", ["$http", function($http) {
   var user = null;
 
+  var login = function(cb) {
+    var request = $http({
```

Comment:
You should not make a request if the user is already logged in:

`if (user) { cb(user); return; }`